certif/app/components/import-candidates.js

```diff
@@ -30,25 +30,19 @@ export default class ImportCandidates extends Component {
 
   _handleErrorMessage(errorResponse) {
     const error = errorResponse?.errors?.[0];
-    if (error?.code) {
-      return this.intl.t(`common.api-error-messages.${error.code}`);
-    }
+    const errorPrefix = this.intl.t('pages.sessions.import.candidates-list.import-fail-prefix', { htmlSafe: true });
 
-    const errorPrefix = 'Aucun candidat n’a été importé. <br>';
-    let errorMessage = `${errorPrefix} Veuillez réessayer ou nous contacter via le formulaire du centre d'aide`;
+    if (error?.code) {
+      if (error?.meta?.line) {
```

Comment:
Pinaillage: si le `if` précèdent est passé, error existe

```suggestion
      if (error.meta?.line) {
```